addons/payment/tests/test_payment_provider.py

```diff
@@ -226,3 +227,112 @@ def test_provider_not_compatible_with_express_checkout(self):
             self.company.id, self.partner.id, self.amount, is_express_checkout=True
         )
         self.assertNotIn(self.provider, compatible_providers)
+
+    def test_payment_provider_availability_report(self):
```

Comment:
Seeing the next test, is this one still needed? Or as for the payment provider, only one where everything is tested is enough?

Reply:
Yes, you are right, forgot to remove it.